docs/attribution.mdx

```diff
@@ -16,62 +16,102 @@ The Overture Maps Foundation does not require text attribution or an OMF logo on
 ### Addresses
 **License for theme:** The addresses data comes from a variety of sources. All carry permissive open licenses. Some have special terms or require attribution. These terms are outlined below.
 #### Australia
-- G-NAF © [Geoscape Australia](https://geoscape.com.au/) licensed by the Commonwealth of Australia. Available under the [Open Geo-coded National Address File End User Licence Agreement](https://geoscape.com.au/wp-content/uploads/2022/08/EULA-G-NAF-Core-1.pdf). Distributed by OpenAddresses. Accessed: 2024-10-13.
+- G-NAF © [Geoscape Australia](https://geoscape.com.au/) licensed by the Commonwealth of Australia. Available under the [Open Geo-coded National Address File End User Licence Agreement](https://geoscape.com.au/wp-content/uploads/2022/08/EULA-G-NAF-Core-1.pdf). Distributed by OpenAddresses. Accessed: 2024-11-10.
```

Comment:
Looks like this license page is now at: https://geoscape.com.au/wp-content/uploads/2024/08/EULA-G-NAF-Core-1.pdf